### PR TITLE
fix(test): added unit test for area reducer

### PR DESCRIPTION
**Pull request by vikram-raj**
<strong>Note: If there are pending changes to the PR, prefix the PR title with "WIP" and add the label "DO NOT MERGE"</strong>

### Mandatory
 - [ ] What does this PR do? 
Added unit tests for area reducer
- [ ] What issue/task does this PR references?
https://openshift.io/openshiftio/openshiftio/plan/detail/2282
- [ ] Are the tests Included?
Yes

### CODE REVIEW

File: src/app/reducers/area.reducer.spec.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import { AreaReducer } from './area.reducer';
+import { initialState as AreaInitialState, AreaState } from './../states/area.state';
+import * as AreaActions from './../actions/area.actions';
+import { AreaUI } from './../models/area.model';
+export type Action = AreaActions.All;
+
+describe('AreaReducer', () => {
+  describe('undefined actions', () => {
```
Comment (jarifibrahim) [inline src/app/reducers/area.reducer.spec.ts]: @vikram-raj Why a `describe` block inside `describe` block?
The usual pattern is 
```
describe
... it
... it
... it
```

File: src/app/reducers/area.reducer.spec.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { AreaReducer } from './area.reducer';
+import { initialState as AreaInitialState, AreaState } from './../states/area.state';
+import * as AreaActions from './../actions/area.actions';
+import { AreaUI } from './../models/area.model';
+export type Action = AreaActions.All;
+
+describe('AreaReducer', () => {
+  it('AreaReducer: undefined action should return the default state', () => {
```
Comment (jarifibrahim) [inline src/app/reducers/area.reducer.spec.ts]: Please reword this to `Undefined action should return the default state`.
The logs would automatically show `AreaReducer:  Undefined action should return the default state`

File: src/app/reducers/area.reducer.spec.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import { AreaReducer } from './area.reducer';
+import { initialState as AreaInitialState, AreaState } from './../states/area.state';
+import * as AreaActions from './../actions/area.actions';
+import { AreaUI } from './../models/area.model';
+export type Action = AreaActions.All;
+
+describe('AreaReducer', () => {
```
Comment (jarifibrahim) [inline src/app/reducers/area.reducer.spec.ts]: Please change to `AreaReducer :`

File: src/app/reducers/area.reducer.spec.ts (hunk 2 above)
Comment (jarifibrahim) [inline src/app/reducers/area.reducer.spec.ts]: Please reword this to `Undefined action should return the default state`.
The logs would automatically show `AreaReducer:  Undefined action should return the default state`

File: src/app/reducers/area.reducer.spec.ts (hunk 3 above)
Comment (vikram-raj) [reply src/app/reducers/area.reducer.spec.ts]: Fixed https://github.com/fabric8-ui/fabric8-planner/pull/2630/commits/1274d2632d41adc040909755d832b507fbd121e3

File: src/app/reducers/area.reducer.spec.ts (hunk 2 above)
Comment (vikram-raj) [reply src/app/reducers/area.reducer.spec.ts]: Fixed https://github.com/fabric8-ui/fabric8-planner/pull/2630/commits/1274d2632d41adc040909755d832b507fbd121e3

**Comment by vikram-raj** [comment]
> You could send an invalid payload and then check the state.

@jarifibrahim Sending invalid payload to actions is not possible because an action only accepts the payload if it is a valid payload otherwise developer will get typescript error.

**Comment by jarifibrahim** [comment]
>@jarifibrahim Sending invalid payload to actions is not possible because an action only accepts the payload if it is a valid payload otherwise developer will get typescript error.

Okay
